exp/test_cases/simple_clouds/socrates_simcloud.py

```diff
@@ -0,0 +1,325 @@
+# diag.add_field('strat_cloud', 'lts', time_avg=True)
+# diag.add_field('strat_cloud', 'ELF', time_avg=True)
+# diag.add_field('strat_cloud', 'zlcl', time_avg=True)
+# diag.add_field('strat_cloud', 'z700', time_avg=True)
+# diag.add_field('strat_cloud', 'gamma700', time_avg=True)
+# diag.add_field('strat_cloud', 'gamma_DL', time_avg=True)
+# diag.add_field('strat_cloud', 'theta', time_avg=True)
+# diag.add_field('strat_cloud', 'dthdp', time_avg=True)
+# diag.add_field('strat_cloud', 'beta1', time_avg=True)
+# diag.add_field('strat_cloud', 'beta2', time_avg=True)
+# diag.add_field('strat_cloud', 'zinv', time_avg=True)
+# diag.add_field('strat_cloud', 'alpha', time_avg=True)
+# diag.add_field('strat_cloud', 'DS', time_avg=True)
+# diag.add_field('strat_cloud', 'IS', time_avg=True)
+
+exp.diag_table = diag
+
+# Empty the run directory ready to run
+exp.clear_rundir()
+
+exp.inputfiles = [os.path.join(GFDL_BASE, 'input/rrtm_input_files/ozone_1990.nc'),
+                  os.path.join(base_dir,  'input/era_land_t42_filtered.nc'),
+                  os.path.join(base_dir,  'input/sst_clim_amip.nc'),
+                  os.path.join(base_dir,  'input/siconc_clim_amip.nc')]
+
+# Define values for the 'core' namelist
+exp.namelist = Namelist({
+    'main_nml':{
+        'days'    : 30,
+        'hours'   : 0,
+        'minutes' : 0,
+        'seconds' : 0,
+        'dt_atmos': 720, # 600
+        'current_date': [1,1,1,0,0,0],
+        'calendar': 'thirty_day'
+    },
+
+    'socrates_rad_nml': {
+        'stellar_constant': 1370.,
+        #'lw_spectral_filename': os.path.join(GFDL_BASE, 'src/atmos_param/socrates/src/trunk/data/spectra/ga7/sp_lw_ga7'),
+        #'sw_spectral_filename': os.path.join(GFDL_BASE, 'src/atmos_param/socrates/src/trunk/data/spectra/ga7/sp_sw_ga7'),
+        'lw_spectral_filename': os.path.join(GFDL_BASE, 'src/atmos_param/socrates/src/trunk/data/spectra/ga3_1/sp_lw_ga3_1'),
+        'sw_spectral_filename': os.path.join(GFDL_BASE, 'src/atmos_param/socrates/src/trunk/data/spectra/ga3_1/sp_sw_ga3_0'),
+        'do_read_ozone': True,
+        'ozone_file_name' : 'ozone_1990',
+        'ozone_field_name': 'ozone_1990',
+        'dt_rad': 4320, # 3600,
+        'store_intermediate_rad': True,
+        'chunk_size': 16,
+        'use_pressure_interp_for_half_levels': False,
+        'tidally_locked': False,
+    },
+
+    'idealized_moist_phys_nml': {
+        'do_damping': True,
+        'turb': True,
+        'mixed_layer_bc': True,
+        'do_virtual': False,
+        'do_simple': True,
+        'roughness_mom'  : 3.21e-05,
+        'roughness_heat' : 3.21e-05,
+        'roughness_moist': 3.21e-05,
+        'two_stream_gray': False,     # Use the grey radiation scheme
+        'do_socrates_radiation': True,
+        'convection_scheme': 'SIMPLE_BETTS_MILLER', # Use simple Betts miller convection
+        'do_cloud_simple': True, # Turn on the cloud scheme switch
+        'land_option': 'input',
```

Comment:
I could possibly include a separate AMIP example test case. I kept the level of complexity with the continents as this is what is used in Liu et al. 2021.

Reply:
It is probably more work than is needed here. If you plan to run an AMIP case, I would suggest you include it in a future PR so that it is included, but if not, disregard my comment.

Reply:
Yes, let's save it for a future PR.